be/src/exec/pipeline/pipeline_driver_queue.h

```diff
@@ -18,6 +18,22 @@ namespace pipeline {
 class DriverQueue;
 using DriverQueuePtr = std::unique_ptr<DriverQueue>;
 
+class SubQuerySharedDriverQueue {
```

Comment:
What's the meaning of `SubQuery` ?

Reply:
it be used for Multi-level Feedback Queue